js_modules/dagit/packages/core/src/workers/runLogs/runLogs.worker.ts

```diff
@@ -0,0 +1,42 @@
+import type {Message} from './runLogs';
+
+type SHUTDOWN = {
+  type: 'SHUTDOWN';
+  staticPathRoot?: undefined;
+};
+
+self.addEventListener('message', ({data}: {data: Message | SHUTDOWN}) => {
+  if (data.staticPathRoot) {
+    __webpack_public_path__ = data.staticPathRoot;
+  }
```

Comment:
Is it possible that there is no `staticPathRoot` defined? I feel like we probably always have something, even if it's just `/`, right?

Reply:
We always send this first so it will always be defined.